package.json

```diff
@@ -1,14 +1,12 @@
 {
-  "name": "safe-token",
+  "name": "@safe-global/safe-token",
   "version": "1.0.0",
   "description": "Repository for Safe token related contracts",
   "homepage": "https://www.safe.global",
-  "license": "GPL-3.0",
+  "license": "LGPL-3.0",
   "files": [
     "contracts",
     "dist",
```

Comment:
I don't think we need to include the `dist` directory. There are no TS libraries that we actually export with this project IMO.